department-of-veterans-affairs/vets-website
# Add support for Cypress local-reporting

## Summary

- Add support for local reports-generation when running Cypress tests locally:
  - Create local-reports Cypress config
  - Create Node-script(s) to run Cypress with local-reporting
- Team: Veteran Facing Forms

## Related issue(s)

- _Link to ticket created in va.gov-team-forms repo_
  - department-of-veterans-affairs/va.gov-team-forms#936
  - department-of-veterans-affairs/va.gov-team-forms#909

## Testing done

- Local testing was successful &mdash; mochawesome reports generated.

### Local testing procedure

1.  Start your app: `yarn watch --env entry=my-entry-name`
2. In separate CLI window/tab, run the script: `yarn cy:run:localreports my-app-folder`
3. Check report-files generated in `/mochawesome-report` project-root folder

## What areas of the site does it impact?

None; this is a vets-website qa-workflow enhancement.
